swarm_node.py

```diff
@@ -126,10 +127,13 @@ def send_transfer(tag, messages, address, values, dict_tips, debug=0):
     elapsed_pow = 0
     time_start_pow = time.time()
     for tx_tryte in trytes:
-        # TODO: Timestamp
-        # timestamp = None
-        # timestamp_lower_bound = None
-        # timestamp_upper_bound = None
+        # Attachment timestamp insert
+        timestamp = TryteString.from_trits(
+            trits_from_int(int(round(time.time() * 1000)), pad=27))
```

Comment:
Follow consistent coding style. Be aware of the comma.

Reply:
Why `round` is invoked? Can you simplify the expression?

Reply:
Sorry, `round` is redundant here. As of coding style, doesn't the line break meet PEP8?